## 05_fix_examplefiles.diff by <danai@debian.org>
##
## Fix examples/CJK.tex: firstly, change from CJK to CJKutf8, and
## secondly, change from cyberb to cyberbit.

--- cjk-20060714/examples/UTF8.tex	2006-07-22 22:10:53.000000000 +0200
+++ new/examples/UTF8.tex	2006-09-05 10:30:33.150336108 +0200
@@ -7,7 +7,7 @@
 
 \documentclass[12pt]{article}
 
-\usepackage{CJK}
+\usepackage{CJKutf8}
 \usepackage[T1]{fontenc}
 
 
@@ -23,5 +23,5 @@
 % we want the Unicode font for normal text also
 \DeclareFontFamily{T1}{song}{}
-\DeclareFontShape{T1}{song}{m}{n}{<-> cyberb00}{}
+\DeclareFontShape{T1}{song}{m}{n}{<-> cyberbit00}{}
 \renewcommand\rmdefault{song}
 
